src/util.h

```diff
@@ -67,6 +67,14 @@ extern "C" {
 
 #define COMPILE_ERROR_ON(cond) ((void)sizeof(char[(cond) ? -1 : 1]))
 
+#define pf_always_inline __attribute__((always_inline)) inline
```

Comment:
The point of these macros is to define them in a way that is as far from compiler as possible, because on different compiler the name of an attribute can be different or the syntax will look differently.

And even in this case it can't work - there's no way to include "inline" in your definition of always_inline attribute.